libavfilter/vf_signature.c

```diff
@@ -669,7 +681,13 @@ static int compare_signbuffer(uint8_t* signbuf1, int len1, uint8_t* signbuf2, in
         .streamcontexts = scontexts
     };
     if (binary_import(signbuf1, len1, &scontexts[0]) < 0 || binary_import(signbuf2, len2, &scontexts[1]) < 0) {
-        av_log(NULL, AV_LOG_ERROR, "Could not create StreamContext from binary data\n");
+        if(!scontexts[0].coarsesiglist) {
```

Comment:
```suggestion
        if(scontexts[0].coarsesiglist) {
```
Probably intended to be that way?